apps/testkit.net/Testkit.Net.Core/Server.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using Newtonsoft.Json;
+using System.Threading.Tasks;
+
+
+namespace Testkit.Net.Core
+{
+    internal class TestSpec
+    {
+        [JsonProperty("test")]
+        internal string Test { get; set; }
+    }
+
+    public class Server
+    {
+        private readonly HttpListener _listener;
+
+        public Server(int port)
+        {
+            _listener = new HttpListener(IPAddress.Parse("127.0.0.1"), port);
```

Comment:
Correct. However, the REST server is a work in progress and is not being used at the moment. It should not block the PR in my opinion.